pages/home/HomePage.jsx

```diff
@@ -36,26 +33,18 @@ var HomePage = React.createClass({
     fixLocation(this.context.location);
   },
   componentDidMount: function() {
-    // auto pops up the Pledge modal if the user is visiting
-    // the homepage for the first time
-    var disableModal = "disableAutoPledgeModal";
-    if (!localStorage[disableModal]) {
-      this.handlePledgeBtnClick();
-      localStorage.setItem(disableModal, "disabled");
-    }
-
-    if (this.context.location.search.pledge === "thanks") {
+    if (this.context.location.search.signup === "thanks") {
       this.props.showModal(ThankYouModal, {
         hideModal: this.props.hideModal
       });
       // Optimizely conversion tracking
       window.optimizely = window.optimizely || [];
-      window.optimizely.push(['trackEvent', 'pledgeFormSubmitted']);
+      window.optimizely.push(['trackEvent', 'NewsletterFormSubmitted']);
     }
   },
-  handlePledgeBtnClick: function() {
-    ga.event({ category: 'Clicked Home CTA', action: 'Pledge to Teach' });
-    this.props.showModal(ModalPledge, {
+  handleEmailBtnClick: function() {
+    ga.event({ category: 'Clicked Home CTA', action: 'Get email updates' });
```

Comment:
nitpicking - but can we change the lettercasing here: "Get Email Updates" and maybe center the header?
like

<img width="598" alt="screen shot 2016-02-25 at 7 19 11 pm" src="https://cloud.githubusercontent.com/assets/2896608/13342273/c85a426a-dbf4-11e5-86c7-fe56ec966391.png">

/cc @sabrinang @kristinashu